test/dom/index.js

```diff
@@ -28,24 +68,168 @@ describe('dom', () => {
+      x: 20,
+      y: 30,
+      width: 400,
+      height: 200,
+      top: -300,
+      right: 900,
+      bottom: 20,
+      left: 500,
+    };
+
+    const node = {
+      getBoundingClientRect() {
+        return data;
+      },
+    };
+
+    const spy = sinon.spy(node, 'getBoundingClientRect');
+
+    assert.deepEqual(screenPosition(node), data, 'returns correct object from method');
+    assert.isTrue(spy.called);
+  });
+
+  it('size', () => {
+    const node = {
+      offsetWidth: 100,
+      offsetHeight: 200,
+    };
+
+    assert.deepEqual(size(node), { width: 100, height: 200 }, 'correctly changes offset w/h to w/h');
+  });
+
+  it('screenSize', () => {
+    const node = {
+      document: {
+        documentElement: {
+          clientWidth: 1200,
+          clientHeight: 800,
+        },
+      },
+    };
+
+    assert.deepEqual(screenSize(node), { width: 1200, height: 800 }, 'correctly extracts document sizes');
+  });
+
+  it('preventDefault', () => {
+    const event = {
+      preventDefault() {
+        return 'prevented default';
+      },
+    };
+
+    const spy = sinon.spy(event, 'preventDefault');
+
+    preventDefault(event);
+    assert.isTrue(spy.called);
+  });
+
+  it('stopPropagation', () => {
+    const event = {
+      stopPropagation() {
+        return 'stopped propagation';
+      },
+    };
+
+    const spy = sinon.spy(event, 'stopPropagation');
+
+    stopPropagation(event);
+    assert.isTrue(spy.called);
+  });
+
+  it('stopEvent', () => {
+    const event1 = {
+      preventDefault() {
+        return 'prevented default';
+      },
+      stopPropagation() {
+        return 'stopped propagation';
+      },
+    };
+    const event2 = {
+      preventDefault() {
+        return 'prevented default';
+      },
+    };
+    const event3 = {
+      stopPropagation() {
+        return 'stopped propagation';
+      },
+    };
+
+    const spy1 = sinon.spy(event1, 'stopPropagation');
+    const spy2 = sinon.spy(event1, 'preventDefault');
+
+    stopEvent(event1);
+    assert.isTrue(spy1.called);
+    assert.isTrue(spy2.called);
+    // Tests for presence of methods
+    assert.throws(() => stopEvent(event2), Error);
+    assert.throws(() => stopEvent(event3), Error);
   });
 
   it('eventCoordinates', () => {
     const pageX = 123;
     const eventTouch = { touches: [{ pageX }] };
 
-    assert.isUndefined(utils.eventCoordinates({}, 'pageX').pageX, 'handles non existant props ok');
-    assert.equal(utils.eventCoordinates(eventTouch, 'pageX').pageX, pageX, 'picks touch coords correctly');
-    assert.equal(utils.eventCoordinates(eventTouch, 'pageX').pageX, pageX, 'picks mouse coords correctly');
+    assert.isUndefined(eventCoordinates({}, 'pageX').pageX, 'handles non existant props ok');
+    assert.equal(eventCoordinates(eventTouch, 'pageX').pageX, pageX, 'picks touch coords correctly');
+    assert.equal(eventCoordinates(eventTouch, 'pageX').pageX, pageX, 'picks mouse coords correctly');
```

Comment:
there is a bug in this test, same thing tested twice, it seems last test should receive { pageX } instead of eventTouch